packages/internal/src/build/api.ts

```diff
@@ -72,15 +81,18 @@ export const prebuildFile = (
 export const getBabelPlugins = () => {
   const rwjsPaths = getPaths()
   const plugins = [
```

Comment:
Maybe leave a comment here for the named module syntax?

Something like "Add names to rw built in plugins to ensure settings don't get overwritten by user config?"